### ISSUE TITLE
google_redis_instance region not being read correctly

### ISSUE THREAD

**Issue by rileykarson** [terraform-providers/terraform-provider-google#2132]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment
* If an issue is assigned to the "modular-magician" user, it is either in the process of being autogenerated, or is planned to be autogenerated soon. If an issue is assigned to a user, that user is claiming responsibility for the issue. If an issue is assigned to "hashibot", a community member has claimed the issue already.

### Terraform Version

HEAD

### Affected Resource(s)

* google_redis_instance

### Terraform Configuration Files

TestAccRedisInstance_full

### Debug Output

N/A

### Panic Output

N/A

### Expected Behavior

No diff

### Actual Behavior

```
region:                  "" => "us-central1" (forces new resource)
```

### Steps to Reproduce

`make testacc TEST=./google TESTARGS=' run=TestAccRedisInstance_full'`

### Important Factoids

This test is failing in CI for some other reason and is disguising this error.
This happens only with explicitly defined regions.

### References

N/A

**Comment by ndmckinley** [comment]
Should be fixed by https://github.com/GoogleCloudPlatform/magic-modules/pull/511.

**Comment by rileykarson** [comment]
Closed with #2133
